week-2-practise/level-3/6-map.js

```diff
@@ -11,11 +11,15 @@ function tidyUpString(str) {
 }
 
 function captialise(str) {
-  // complete this function
+  return (
```

Comment:
This does work, but the function name is a bit confusing. We are now calling capitalise and it is tidying up our string!

If we had
`var mentorsTidy = mentors.map(tidyUpString);`
can you think of a way to call `mentorsTidy.map` that would give the results we want?